### DISCUSSION TITLE
Unauthorized key

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2631] [Q&A]
![Screenshot (37)](https://github.com/user-attachments/assets/a451b2b2-985d-4461-9bac-c0467125a4da)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, I didn't use `thirdweb` for my deployment process, so I am unsure what you did wrong. There is a troubleshooting link on the terminal that will take you to thridweb website to find out what was wrong. Please click on the click and see what they have to say.

**Comment by farman1094** [comment]
I didn't use the third web as well. It'ys looking the issue of private key or `secret key`. Check if you are providing the right value or right syntax maybe.
